coordinator/gscoordinator/kubernetes_launcher.py

```diff
@@ -433,6 +437,73 @@ def _create_mars_scheduler(self):
         )
         self._resource_object.append(response)
 
+    # the function is used to inject vineyard as a sidecar container into the workload
+    # 1. add the annotations and labels to the workload for enabling the injection
+    # 2. get the ownerReference of the coordinator
+    # 3. apply the etcd cluster for vineyard sidecar
+    # 4. return the json string of new workload which is injected with vineyard sidecar
+    def _inject_vineyard_as_sidecar(self, workload):
```

Comment:
Could you give a sample output of the injected json? I want to know which portion of the body will be modified and how many components would be added to that. And could the user control the resource, replicas, or even selectors, labels, affinities of the injected items?

Reply:
I think better to give a link to the documentation of vineyardctl.inject.

Reply:
Okay, I will add it later.